tests/sur/quota.js

```diff
@@ -27,26 +43,30 @@ function createBucket(bucket, locked, cb) {
     if (locked) {
         config.ObjectLockEnabledForBucket = true;
     }
-    return s3Client.createBucket(config, (err, data) => {
-        assert.ifError(err);
-        return cb(err, data);
-    });
+    return s3Client.send(new CreateBucketCommand(config)) 
+        .then(data => cb(null, data))
+        .catch(err => {
+            assert.ifError(err);
```

Comment:
I was curious about an assert in a catch and I think this will not work:

```
const assert = require('assert');

describe.only('Array', () => {
  it.only('should return -1 when the value is not present', done => {
    function test(cb) {
      new Promise((_resolve, reject) => {
        reject('C est pas vrai');
      })
        .then(cb)
        .catch(err => {
          assert.ifError(err);
          cb();
        });
    }

    test(done);
  });
});
```

The callback is never called ?